File: solutions/python/bob/1/bob.py
def response(hey_bob):
    
    hey_bob=hey_bob.strip()
    

    if hey_bob.upper()==hey_bob and hey_bob.endswith("?") and any(c.isalpha() for c in hey_bob):
        return "Calm down, I know what I'm doing!"

    if hey_bob.endswith("?"):
        return "Sure."

    if hey_bob.upper()==hey_bob and any(c.isalpha() for c in hey_bob):
        return "Whoa, chill out!"

    if hey_bob.strip()=="":
        return "Fine. Be that way!"
    
    return "Whatever."

File: solutions/javascript/bob/1/bob.js
//
// This is only a SKELETON file for the 'Bob' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const hey = (message) => {
  message = message.trim();
  if (message === ""){
    return "Fine. Be that way!";
  }

  if (message.toUpperCase() == message && (/[A-Za-z]/.test(message))){
    if(message.endsWith("?")){
      return "Calm down, I know what I'm doing!";
    }
    return "Whoa, chill out!";
  }

  if(message.endsWith("?")){
    return "Sure.";
  }
  return "Whatever.";
};
